### ISSUE TITLE
VS Dual System

### ISSUE THREAD

**Issue by alyosha-tas** [TASVideos/BizHawk#1490]
Thinking through natural extensions of recent work on linked play systems, I remembered that Nintendo's arcade VS system has a 'Dual System' variant that currently isn't supported.

It should be possible to implement using the exact same methodology as SubNESHawk / GBHawkLink uses. 

I'm putting it up here as a reminder since I probably won't get to it for some time.

If someone wants to try their hand at it, feel free. It should be a not too difficult exercise in working with BizHawk's core system.

**Comment by adelikat** [comment]
Oh yeah, that.  Is there even an NES emulator that supports this? This could be an innovative feature.  Doing it like DualGB is an interesting idea.

**Comment by alyosha-tas** [comment]
Yes other modern nes emulators support this already. So once again it would just be keeping up with modern developments.

**Comment by YoshiRulz** [comment]
Isn't it just 2 gamepads + 2 screens, sorta like NDS? I very much doubt they'd bother with any sort of networking in an arcade setting.
